Upstream-Status: Inappropriate [configuration]

tinylogin will meet segment fault if compiled by gcc-4.5.0 when enable both
frename-registers and Os options. Use O2 instead.

Signed-off-by: Dongxiao Xu <dongxiao.xu@intel.com>

diff -ruN tinylogin-1.4-orig/Makefile tinylogin-1.4/Makefile
--- tinylogin-1.4-orig/Makefile	2010-09-16 13:02:06.302192295 +0800
+++ tinylogin-1.4/Makefile	2010-09-16 13:02:34.634167846 +0800
@@ -97,9 +97,7 @@
 #--------------------------------------------------------
 
 
-# use '-Os' optimization if available, else use -O2
-OPTIMIZATION = $(shell if $(CC) -Os -S -o /dev/null -xc /dev/null >/dev/null 2>&1; \
-    then echo "-Os"; else echo "-O2" ; fi)
+OPTIMIZATION = -O2
 
 WARNINGS = -Wall
 
